LSP.sublime-settings

```diff
@@ -8,6 +8,22 @@
   // Valid values are "never", "always" and "saved"
   "auto_show_diagnostics_panel": "saved",
 
+  // A dictionary of code action identifiers that should be triggered on save.
+  //
+  // Code action identifiers are not officially standardized so refer to specific
+  // server's documentation on what is supported but `source.fixAll` is commonly
+  // used to apply fix-on-save code action.
+  //
+  // It's also supported to set this in project-specific settings in which case
+  // it will only apply in that specific project. Setting it here will apply globally,
+  // unless overridden by project-specific settings.
+  //
+  // Only "source.*" actions are supported.
+  "lsp_code_actions_on_save": {},
```

Comment:
If you want this to be a view setting, then it should not be here in LSP.sublime-settings. Instead tell users to put it in Preferences.sublime-settings and/or in their "settings" of their .sublime-project and/or in their syntax-specific override settings. It's the same kind of setting as lsp_format_on_save.

Reply:
Actually the idea is that you can set it in LSP.sublime-settings (then it will apply globally) or you can use it in project-specific settings.

The `LspSaveCommand._get_code_actions_on_save` has logic to take all of those into account.

Reply:
But if it's put into Packages/User/Preferences.sublime-settings, then it's also global simply via checking `view.settings().get("lsp_code_actions_on_save")`.

Reply:
But then it makes that setting less discoverable?
Or how would you suggest to handle discoverability then?
I think that only having a mention in documentation is not enough.

Reply:
`lsp_format_on_save` already works the way I describe, why do things differently for `lsp_code_actions_on_save`?

Reply:
I would suggest changing how `lsp_format_on_save` works then maybe.

I'd say it's better this way as I wouldn't even know the `lsp_format_on_save` existed if you wouldn't just tell me.

It's nice to be able to see all settings LSP supports when looking at its preferences.

And also, if it's not in Preferences then PackageDev marks pref as non-existent when user puts it in his global Preferences.

Reply:
I'm okay with changing how `lsp_format_on_save` works, as long as these two settings are consistent.

Reply:
Done